cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -232,6 +233,7 @@ def error_response(message, status, stage):
 @require_GET
 @ensure_csrf_cookie
 @login_required
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
```

Comment:
Can you explain the new policy for the cache?

Reply:
`no-cache` This instructs the browser that it must revalidate with the server every time before using a cached version of the URL.
`no-store` This instructs the browser and other intermediate caches (like CDNs) to never store any version of the file

for further information https://web.dev/http-cache/

Reply:
great 👍 
its good to capture this in the desc as well.